cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/objectstorage/ObjectStorageValidateRequest.java

```diff
@@ -203,6 +219,11 @@ public Builder withBackupLocationBase(String backupLocationBase) {
             return this;
         }
 
+        public Builder withBackupOperationType(String backupOperationType) {
```

Comment:
why are you using String here as parameter?

Reply:
addressed